addons/mail/models/mail_template.py

```diff
@@ -82,6 +82,8 @@ def _get_non_abstract_models_domain(self):
         string='Dynamic Reports',
         domain="[('model', '=', model)]")
     email_layout_xmlid = fields.Char('Email Notification Layout', copy=False)
+    email_notification_allow_header = fields.Boolean('Show Header', default=True)
```

Comment:
if we introduce those, maybe we should think of propagating them through notification layer. note that also we cannot distinguish "I don't want the footer" from "I don't care, just have the default behavior". Some kind of ternary field.

Reply:
So maybe we should just have "force_header" and "force_footer" that would bypass current behavior